app/Document/Paragraph.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Document;
+
+use App\Document\Sentence;
+
+class Paragraph
```

Comment:
What is the benefit of moving to represent the `Paragraph`, `Sentence` and `Word` as objects?